notifier/notifier.go

```diff
@@ -0,0 +1,48 @@
+package notifier
+
+import (
+	"context"
+	"log"
+
+	"github.com/appleboy/gorush/rpc/proto"
```

Comment:
This package has just one Go file and none `gorush` dependencies. Can we include only it in `Gopkg.toml`?

Reply:
isn't it already like this?

```
+  name = "github.com/appleboy/gorush"
+  packages = ["rpc/proto"]
```

i think that most of the deps are from grpc that is also imported one line below

but the same should b doable with regular http client (please see comment above)